accessibility_monitoring_platform/apps/cases/models.py

```diff
@@ -654,9 +684,13 @@ def in_report_correspondence_progress(self) -> Link:
                 label="4-week follow-up to report sent, case needs to progress",
                 url=reverse("cases:edit-report-acknowledged", kwargs={"pk": self.id}),
             )
+
         return Link(
             label="Unknown",
-            url=reverse("cases:manage-contact-details", kwargs={"pk": self.id}),
+            url=reverse(
+                "cases:manage-contact-details",
+                kwargs={"pk": self.id}
+            ),
```

Comment:
This looks like a `black` code formatting change. Lets make sure that we are both using the same version of `black`:

```
black --version
black, 24.10.0 (compiled: yes)
Python (CPython) 3.11.10
```

Are you using a smaller line-length than the default 88?

Reply:
The formatting seems consistent now.